HicServices/RDMP
# Task/RDMP-180 Add instance settings table

Adds the ability to store Key/Value configuration for the RDMP instance.
Adds the instance setting for auto-suggesting project numbers

To test:
There is a new "instance settings" tab under "view" in the top nav bar.
Toggle the project number option then create a new project - the input for project number should/should not be populated based on your selection
